<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EviModelView</class>
 <widget class="QWidget" name="EviModelView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChargePoint</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="leUrl"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="teDisplay"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leTagUID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>FAFAFAFA</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnTouchCard">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>TouchCard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnEna">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReboot">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reboot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPowerError">
       <property name="text">
        <string>PowerError</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="connectorHolder">
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
